frontend/package-lock.json

```diff
@@ -1,4503 +1,9 @@
 {
   "name": "frontend",
   "version": "0.1.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
What version of npm are you running on your machine?

Reply:
Besides that it looks good, just want to clarify before merging to avoid losing frontend dependencies.